### PR TITLE
[ISSUE #5807]Add unit tests for package com.alibaba.nacos.client.utils in nacos

### LINKED ISSUE #5807: Add unit tests for package com.alibaba.nacos.client.utils in nacos 2.0
This is a sub-issue of [ISSUE #5011]

### PR DESCRIPTION
**Pull request by shalk**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

fix #5807

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/test/java/com/alibaba/nacos/client/utils/ParamUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    @Test
+    public void testGetAppKey() {
+        String defaultVal = ParamUtil.getAppKey();
+        Assert.assertEquals("", defaultVal);
+        
+        String expect = "test";
+        ParamUtil.setAppKey(expect);
+        Assert.assertEquals(expect, ParamUtil.getAppKey());
+        
+        ParamUtil.setAppKey(defaultVal);
```
Comment (KomachiSion) [inline client/src/test/java/com/alibaba/nacos/client/utils/ParamUtilTest.java:39]: If assert fail, the reset will invalid.

File: client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+public class TenantUtilTest {
+    
+    @Test
+    public void testGetUserTenantForAcm() {
+        String expect = "test";
+        System.setProperty("acm.namespace", expect);
+        String actual = TenantUtil.getUserTenantForAcm();
+        Assert.assertEquals(expect, actual);
+        System.clearProperty("acm.namespace");
```
Comment (KomachiSion) [inline client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java:32]: If assert fail, the reset will invalid.

you can do this in After method.

Comment (shalk) [reply client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java:32]: ok, i will fix this

**Comment by KomachiSion** [comment]
CI can't pass.

**Comment by shalk** [comment]
> CI can't pass. Please pull new develop branch.

i think the ci issue is fixed by brotherlu-xcq . great job.

i looked into the MemberUtilTest last night, but do not found which thread change the `failcnt`, it is too late 😄

File: client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java (hunk 2 above)
Comment (KomachiSion) [reply client/src/test/java/com/alibaba/nacos/client/utils/TenantUtilTest.java:32]: This case do not fix to clear env in after.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**